openmls/src/group/mls_group/ser.rs

```diff
@@ -25,9 +26,9 @@ pub struct SerializedMlsGroup {
     group_state: MlsGroupState,
 }
 
-impl SerializedMlsGroup {
-    /// Helper method that converts the SerializedMlsGroup to MlsGroup.
-    pub fn into_mls_group(self) -> MlsGroup {
+#[allow(clippy::from_over_into)]
```

Comment:
I think it's because you only want a one way conversion between the serialized representation and the actual, functioning `MlsGroup`. The other way is taken care of by the `serde::Serialize` trait implementation.

We could totally get rid of the manual serde trait impl (in favor of a `From` implementation) but I wanted to keep the changes minimal in order to minimize churn.

In essence this change is just an API rename of `into_mls_group` to `into`.